### PR TITLE
Get bottom drawer background and mailbox title to be in sync with drawer animation

### PR DESCRIPTION
**Pull request by Renzo-Olivares**
This change has the opacity of the bottom drawer background and mailbox title animate based on the drawer animation. It also adds a `FadeThroughAnimation` for when the mailbox title disappears when going from mailbox to mailview.

reply-mailbox-fade-through

### CODE REVIEW

File: lib/home.dart [hunk 1]
```diff
@@ -433,18 +435,20 @@ class _AnimatedBottomAppBar extends StatelessWidget {
...
-                            navigationDestinations[selectedIndex].name,
-                            style: Theme.of(context)
-                                .textTheme
-                                .bodyText1
-                                .copyWith(color: ReplyColors.white50),
-                          ),
+                        _FadeThroughTransitionSwitcher(
+                          fillColor: Colors.transparent,
+                          child: onMailView
+                              ? const SizedBox(height: 0, width: 48)
```
Comment (clocksmith) [inline lib/home.dart:441]: what is `const SizedBox(height: 0, width: 48)` for?

Comment (Renzo-Olivares) [reply lib/home.dart:441]: It's a placeholder for the disappeared state of the mailbox title.

Comment (clocksmith) [reply lib/home.dart:441]: can it just be Container()? why does it need width 48 and height 0?

Comment (Renzo-Olivares) [reply lib/home.dart:441]: I do this to keep the tappable area of the logo the same. If not when the title disappears the tappable area would be smaller, as it only covers the Reply logo and drop arrow.

**Review by rami-a — APPROVED** [review]

**Review by clocksmith — APPROVED** [review]

**Pull request merged**